packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -58,6 +58,16 @@ export class StarkTableComponent implements OnInit, AfterContentInit, AfterViewI
 	 */
 	@Input() public filter: StarkTableFilter;
 
+	/**
+	 * Array of StarkAction objects (see StarkAction docs).
+	 */
+	@Input() public customTableActions?: StarkAction[];
+
+	/**
+	 * Type of StarkAction objects
+	 */
+	@Input() public customTableActionsType: 'regular' | 'alt' | string = 'regular';
```

Comment:
Could you please replace `'` by `"` ?
(Prettier should do it automatically normally 😕)